jenkins.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: jenkins
-  version: "2.463"
```

Comment:
I don't understand this https://github.com/wolfi-dev/os/blob/main/jenkins.yaml#L3. Jenkins is already at `2.464` 👻

Reply:
conflict

Reply:
nvm